### PR TITLE
send_addressモデルのテスト

### PR DESCRIPTION
**Pull request by Nishihatak**
# what
send_addressモデルのテスト

# why
モデルで問題なくバリデーションが効いているのか調べる。

### CODE REVIEW

**Comment by Nishihatak** [comment]
SendAddress
  #create
    False
      presence_all_false
        is invalid without a kanji_last_name
        is invalid without a kanji_first_name
        is invalid without a kana_last_name
        is invalid without a kana_first_name
        is invalid without a post_code
        is invalid without a prefecture
        is invalid without a city
        is invalid without a address
      kanji_last_name
        is invalid with a kanji_last_name only number
      kanji_first_name
        is invalid with a kanji_first_name only number
      kana_last_name
        is invalid with a kana_last_name only alphabet
        is invalid with a kana_last_name only hiragana
        is invalid with a kana_last_name only kanji
      kana_first_name
        is invalid with a kana_first_name only alphabet
        is invalid with a kana_first_name only hiragana
        is invalid with a kana_first_name only kanji
      post_code
        is invalid more than 8-digits-numbers
        is invalid without 7-characters
        is invalid wrongs format
      prefecture_id
        is invalid with a prefecture_id mistake number
        is invalid with a prefecture_id 0
    True
      presence_all_true
        is valid with a all contents
      kanji_last_name
        is invalid with a kanji_last_name only hiragana
        is invalid with a kanji_last_name only katakana
        is invalid with a kanji_last_name only kanji
      kanji_first_name
        is invalid with a kanji_first_name only hiragana
        is invalid with a kanji_first_name only katakana
        is invalid with a kanji_first_name only kanji
      kana_last_name
        is invalid with a kana_last_name only katakana
      kana_first_name
        is invalid with a kana_first_name only katakana
      post_code
        is invalid with a post_code
      prefecture_id
        is invalid with a only integer
      city
        is invalid with a city only hiragana
        is invalid with a city only katakana
        is invalid with a city only kanji
        is invalid with a city only alphabet
      address
        is invalid with a address use hiragana
        is invalid with a address use katakana
        is invalid with a address use kanji
        is invalid with a address only alphabet

File: app/models/send_address.rb [hunk 1]
```diff
@@ -13,9 +13,9 @@ class SendAddress < ApplicationRecord
   validates :kana_first_name,   presence: true
   validates :kana_first_name,   kana: true, allow_blank: true
   validates :post_code,         presence: true
-  validates :post_code,         format: {with: /\A\d{3}[-]\d{4}$|^\d{3}[-]\d{2}$|^\d{3}\z/,message:"フォーマットが不適切です"}, allow_blank: true
+  validates :post_code,         format: {with: /\A\d{3}[-]\d{4}$\z/,message:"のフォーマットが不適切です"}, allow_blank: true
   validates :prefecture_id,     presence: true
-  validates :prefecture_id,     format: {with: /[1-47]/,message:"を入力してください"}, allow_blank: true
+  validates :prefecture_id,     format: {with: /\A([1-9]|0[1-9]|[1-3][0-9]|4[0-7])\z/,message:"を入力してください"}, allow_blank: true
   validates :city,              presence: true
```
Comment (tatsuya551) [inline app/models/send_address.rb]: ０から９までのフォーマットを訂正して下さい。

Comment (tatsuya551) [reply app/models/send_address.rb]: LGTMです!!

**Comment by tatsuya551** [comment]
LGTMです！！

**Comment by Nishihatak** [comment]
[Image from Gyazo](https://gyazo.com/7566dabfdf193530036bc5290b3652e4)

**Pull request merged**